components/cam/bld/namelist_files/use_cases/20TR_E3SMv1_superfast_ar5-emis.xml

```diff
@@ -0,0 +1,173 @@
+<dms_emis_file	      >atm/cam/chem/trop_mozart_aero/emis/DMSflux.1850-2100.1deg_latlon_conserv.POPmonthlyClimFromACES4BGC_c20160727.nc</dms_emis_file>
+<so2_emis_file	      >atm/cam/chem/trop_mozart_aero/emis/DECK_ne30/cmip6_mam4_so2_surf_1850-2014_c180205.nc </so2_emis_file>
+<bc_a4_emis_file      >atm/cam/chem/trop_mozart_aero/emis/DECK_ne30/cmip6_mam4_bc_a4_surf_1850-2014_c180205.nc </bc_a4_emis_file>
+<mam7_num_a1_emis_file>atm/cam/chem/trop_mozart_aero/emis/DECK_ne30/cmip6_mam4_num_a1_surf_1850-2014_c180205.nc </mam7_num_a1_emis_file> 
+<num_a2_emis_file     >atm/cam/chem/trop_mozart_aero/emis/DECK_ne30/cmip6_mam4_num_a2_surf_1850-2014_c180205.nc </num_a2_emis_file>
+<mam7_num_a3_emis_file>atm/cam/chem/trop_mozart_aero/emis/DECK_ne30/cmip6_mam4_num_a4_surf_1850-2014_c180205.nc </mam7_num_a3_emis_file> <!-- This is to set num_a4 emissions -->
+<pom_a4_emis_file     >atm/cam/chem/trop_mozart_aero/emis/DECK_ne30/cmip6_mam4_pom_a4_surf_1850-2014_c180205.nc </pom_a4_emis_file>
+<so4_a1_emis_file     >atm/cam/chem/trop_mozart_aero/emis/DECK_ne30/cmip6_mam4_so4_a1_surf_1850-2014_c180205.nc </so4_a1_emis_file>
+<so4_a2_emis_file     >atm/cam/chem/trop_mozart_aero/emis/DECK_ne30/cmip6_mam4_so4_a2_surf_1850-2014_c180205.nc </so4_a2_emis_file>
+<ch2o_emis_file    >atm/cam/chem/1850-2000_emis/IPCC_emissions_houw_CH2O_1850-2000_1.9x2.5.c090728.nc</ch2o_emis_file>
+<co_emis_file      >atm/cam/chem/1850-2000_emis/IPCC_emissions_houw_CO_1850-2000_1.9x2.5.c090728.nc  </co_emis_file>
+<nox_emis_file     >atm/cam/chem/1850-2000_emis/IPCC_emissions_houw_NOx_1850-2000_1.9x2.5.c090728.nc </nox_emis_file>
+<isop_emis_file    >atm/cam/chem/1850-2000_emis/IPCC_emissions_houw_ISOP_1850-2000_1.9x2.5.c090827.nc</isop_emis_file>
+
+<!-- 1990? aircraft emis: CO and NO  -->
+<airpl_emis_file>atm/waccm/emis/emis.air.nc</airpl_emis_file>  <!-- WARNING: emis.air.nc is NOT time-varying -->
+
+<!-- Prescribed oxidants for aerosol chemistry.   Ozone is from CMIP6 input4MIPS file -->
+<!-- WARNING: THIS ISN'T THE FULL HISTORICAL TIME-SERIES --> 
+<tracer_cnst_type    >CYCLICAL</tracer_cnst_type>
+<tracer_cnst_cycle_yr>1995</tracer_cnst_cycle_yr>
+<tracer_cnst_file    >ch4_oxid_1.9x2.5_L26_1990-1999clim.c090804.nc</tracer_cnst_file>
+<tracer_cnst_filelist>''</tracer_cnst_filelist>
+<!-- <tracer_cnst_datapath>atm/cam/chem/trop_mozart_aero/oxid</tracer_cnst_datapath>  -->
+<!-- <tracer_cnst_specifier>'NO3'</tracer_cnst_specifier>                             -->
+
+<!-- <tracer_cnst_filelist>this_field_is_not_used</tracer_cnst_filelist> -->
+
+<!-- prescribed methane  -->
+<!-- WARNING: THIS ISN'T THE FULL HISTORICAL TIME-SERIES --> 
+<prescribed_ghg_file      >ch4_oxid_1.9x2.5_L26_1990-1999clim.c090804.nc</prescribed_ghg_file>
+<prescribed_ghg_datapath  >atm/cam/chem/methane</prescribed_ghg_datapath>
+<prescribed_ghg_type      >CYCLICAL</prescribed_ghg_type>
+<prescribed_ghg_cycle_yr  >1995</prescribed_ghg_cycle_yr>
+<prescribed_ghg_filelist  >''</prescribed_ghg_filelist>
```

Comment:
What about other GHGs that are not in the super-fast chemistry? Do we need to prescribe them as well?

Reply:
The other GHGs are specified by global concentration for the CMIP6 compsets, which happens in a different part of the defaults.

Reply:
Thanks, Philip. Do you know what that part is called? I've had this question for some time. I can check it, just need some key words.

Reply:
I think it happens the build-namelist script, which is in perl and is ugly, with messy logic.  The whole system is rickety and fragile, IMHO.  The reason for having it in the build-namelist is because it is a fundamental part of the superfast_mam4 mechanism, and so is not really an option that a user should be allowed to change, although they could override it in user_namelist.  

The lines commented out in the PR use_case file are there because I was using them to construct the chemistry mechanism.   After having this conversation, I think it would be best to delete those commented out lines.   Do you agree?

Reply:
Sounds good to me to delete those commented lines if they are not helpful.